splitter/splitter.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/http;
+import ballerina/mime;
+import ballerina/url;
+
+type ReminderRequest record {
+    string date;
+    Event[] events;
+};
+
+type Event record {|
+    string eventName;
+    Attendee[] attendees;
+|};
+
+type Attendee record {|
+    string name;
+    string number;
+|};
+
+const FROM_NO = "+15005550006";
+const TWILIO_SID = "VAC1829a53d52f41b4b2b1cc003c0026aa8";
+const  API_VERSION = "2010-04-01";
+const CHARSET = "utf-8";
+
+final http:Client twilioClient = check new ("http://api.twilio.com.balmock.io");
+
+service /api/v1 on new http:Listener(8080) {
+    resource function post reminders(ReminderRequest request) returns error? {
+        from Event event in request.events
+            do {
+                _ = from Attendee attendee in event.attendees
+                select sendReminder(attendee, event.eventName, request.date);
+            };
```

Comment:
Having select clause is not correct.
```ballerina
from Event event in request.events
from Attendee attendee in event.attendees
do {
sendReminder(attendee, event.eventName, request.date);
};
```
Will this work?

Reply:
If this is not working we have to use two do statements.

Reply:
The given code doesn't work as variable assignment is required. Will go ahead with the two `do` statements.

Reply:
This works, will go ahead with it.